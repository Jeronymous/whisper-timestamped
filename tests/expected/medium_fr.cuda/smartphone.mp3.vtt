WEBVTT

00:00.320 --> 00:03.680
C'est évident ce que dit Nicolas, mais je ne me l'étais jamais formulé comme ça.

00:04.080 --> 00:08.020
Ce qui fait la force du smartphone, c'est pas seulement l'accumulation des fonctions,

00:08.320 --> 00:10.920
mais la manière dont elles interagissent entre elles.

00:10.960 --> 00:12.960
Ce qui dit d'ailleurs sur la photo, c'est hyper convaincant.

00:13.320 --> 00:16.300
Alors évidemment, il faudrait ajouter les interfaces.

00:16.300 --> 00:19.380
L'écran tactile a été beaucoup très souvent mentionné.

00:19.940 --> 00:25.610
Mais bon, il faut dire qu'il profite aussi de 20 ans pendant lesquels les ordinateurs nous ont appris à cliquer sur des icônes.

00:25.610 --> 00:30.700
Sauf que le smartphone ajoute le toucher, ce qui rend le contact plus direct, plus sensible.

00:31.020 --> 00:37.830
Et puis évidemment, il faudrait parler aussi des applications qui permettent de contourner le côté touffu de la navigation web pour aller directement au but.

00:37.830 --> 00:46.620
Bref, tout ça, ce sont les conditions qui permettent de créer cet objet dont Nicolas dit qu'il est vraisemblablement inédit dans l'histoire de l'humanité.

00:46.760 --> 00:48.800
Mais ça, ça soulève une autre interrogation.

00:49.140 --> 00:55.460
Est-ce que le fait que cet objet soit inédit induit que notre rapport à lui est aussi un rapport inédit?

00:55.580 --> 01:03.240
Je veux dire, est-ce que le rapport qu'on a au smartphone est comparable à celui qu'on entretenait à d'autres objets techniques comme la voiture ou le téléphone?

01:03.360 --> 01:06.640
Il n'y a pas d'équivalent en fait.

01:06.880 --> 01:11.520
Et donc cette espèce de nouveauté dans la relation à l'objet, c'est fascinant et terrifiant.

01:11.620 --> 01:16.520
Parce qu'on a l'impression, comme le disent les utilisateurs et les services, d'être dépendants de cet objet,

01:16.860 --> 01:23.360
d'induire en fait une espèce de relation, de médiation avec le monde qui rend de l'ampleur et qui amène aussi à des formes de rejet.

01:23.900 --> 01:27.660
Donc, à objet inédit, rapport inédit.

01:28.020 --> 01:35.120
Et ce rapport, si j'en crois Nicolas, serait caractérisé par un mélange de dépendance et de rejet.

01:35.760 --> 01:42.840
Bon, en vrai, il faudrait remonter très très finement toute l'histoire des objets techniques et de leur insertion dans nos vies

01:42.880 --> 01:45.740
pour déterminer si ce rapport est totalement inédit.

01:46.040 --> 01:49.440
Mais j'ai l'impression comme ça que Nicolas ne se trompe pas vraiment.

01:49.880 --> 01:55.000
Pour autant que je sache, il y a eu plein de discussions autour de la voiture ou même du téléphone.

01:55.480 --> 01:59.840
Mais la dépendance n'était pas du même ordre. Donc le rejet non plus n'était pas du même ordre.

02:00.000 --> 02:03.100
On peut adorer sa bagnole, en avoir besoin pour plein de choses.

02:03.260 --> 02:06.540
Et bien, le soir, quand on va se coucher, on la laisse.

02:06.980 --> 02:10.220
On ne l'a pas dans la main quand on est au lit, on ne l'emmène pas au chiottes.

02:10.860 --> 02:16.880
On pouvait être énervé par son môme qui occupait la ligne de téléphone pendant une heure chaque soir pour discuter avec un copain.

02:17.280 --> 02:21.780
Mais ça ne ressemblait pas à ce qu'on peut ressentir à voir ce même môme aujourd'hui,

02:22.420 --> 02:26.320
continuellement avec son smartphone dans la main, comme si c'était une sorte de pacemaker externe,

02:26.400 --> 02:28.840
comme si le lâcher allait entraîner sa mort immédiate.

02:29.020 --> 02:31.960
Bon, je dis ça pour le môme, mais c'est évidemment valable pour nous aussi.

02:32.560 --> 02:38.400
Donc, rapport inédit. D'accord. Mais pourquoi a-t-on l'impression qu'on n'en sortira jamais?

02:38.900 --> 02:45.300
Est-ce qu'il faut en remettre la faute sur les gens qui ont créé cet outil merveilleux et diabolique, et diabolique parce que merveilleux?

02:46.820 --> 02:48.880
Les économistes parlent de dépendance du sentier.

02:48.880 --> 02:57.900
C'est l'idée qu'on est sur un sentier qui a été établi, soit volontairement en marchant dessus, soit en définissant des bornes, en définissant une signalétique.

