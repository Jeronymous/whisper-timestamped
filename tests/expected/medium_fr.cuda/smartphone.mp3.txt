C'est évident ce que dit Nicolas, mais je ne me l'étais jamais formulé comme ça.
Ce qui fait la force du smartphone, c'est pas seulement l'accumulation des fonctions,
mais la manière dont elles interagissent entre elles.
Ce qui dit d'ailleurs sur la photo, c'est hyper convaincant.
Alors évidemment, il faudrait ajouter les interfaces.
L'écran tactile a été beaucoup très souvent mentionné.
Mais bon, il faut dire qu'il profite aussi de 20 ans pendant lesquels les ordinateurs nous ont appris à cliquer sur des icônes.
Sauf que le smartphone ajoute le toucher, ce qui rend le contact plus direct, plus sensible.
Et puis évidemment, il faudrait parler aussi des applications qui permettent de contourner le côté touffu de la navigation web pour aller directement au but.
Bref, tout ça, ce sont les conditions qui permettent de créer cet objet dont Nicolas dit qu'il est vraisemblablement inédit dans l'histoire de l'humanité.
Mais ça, ça soulève une autre interrogation.
Est-ce que le fait que cet objet soit inédit induit que notre rapport à lui est aussi un rapport inédit?
Je veux dire, est-ce que le rapport qu'on a au smartphone est comparable à celui qu'on entretenait à d'autres objets techniques comme la voiture ou le téléphone?
Il n'y a pas d'équivalent en fait.
Et donc cette espèce de nouveauté dans la relation à l'objet, c'est fascinant et terrifiant.
Parce qu'on a l'impression, comme le disent les utilisateurs et les services, d'être dépendants de cet objet,
d'induire en fait une espèce de relation, de médiation avec le monde qui rend de l'ampleur et qui amène aussi à des formes de rejet.
Donc, à objet inédit, rapport inédit.
Et ce rapport, si j'en crois Nicolas, serait caractérisé par un mélange de dépendance et de rejet.
Bon, en vrai, il faudrait remonter très très finement toute l'histoire des objets techniques et de leur insertion dans nos vies
pour déterminer si ce rapport est totalement inédit.
Mais j'ai l'impression comme ça que Nicolas ne se trompe pas vraiment.
Pour autant que je sache, il y a eu plein de discussions autour de la voiture ou même du téléphone.
Mais la dépendance n'était pas du même ordre. Donc le rejet non plus n'était pas du même ordre.
On peut adorer sa bagnole, en avoir besoin pour plein de choses.
Et bien, le soir, quand on va se coucher, on la laisse.
On ne l'a pas dans la main quand on est au lit, on ne l'emmène pas au chiottes.
On pouvait être énervé par son môme qui occupait la ligne de téléphone pendant une heure chaque soir pour discuter avec un copain.
Mais ça ne ressemblait pas à ce qu'on peut ressentir à voir ce même môme aujourd'hui,
continuellement avec son smartphone dans la main, comme si c'était une sorte de pacemaker externe,
comme si le lâcher allait entraîner sa mort immédiate.
Bon, je dis ça pour le môme, mais c'est évidemment valable pour nous aussi.
Donc, rapport inédit. D'accord. Mais pourquoi a-t-on l'impression qu'on n'en sortira jamais?
Est-ce qu'il faut en remettre la faute sur les gens qui ont créé cet outil merveilleux et diabolique, et diabolique parce que merveilleux?
Les économistes parlent de dépendance du sentier.
C'est l'idée qu'on est sur un sentier qui a été établi, soit volontairement en marchant dessus, soit en définissant des bornes, en définissant une signalétique.
